test/controllers/teams_controller_test.rb

```diff
@@ -181,5 +181,14 @@ class TeamsControllerTest < ActionController::TestCase
     assert_equal I18n.t('teams.in_top_ten'), flash[:alert]
   end
 
+  test 'can not update team when in top ten' do
```

Comment:
Could you change this to say `cannot update team when game is open and team in top ten`